src/pages/Yeonju/Main/Components/Upload/Upload.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import Comment from '../Comment/Comment';
+
+import './Upload.scss';
+
+import {
+  faPaperPlane,
+  faHeart,
+  faComment,
+  faBookmark,
+  faSmile,
+} from '@fortawesome/free-regular-svg-icons';
+
+class Upload extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      commentValue: '',
+      commentList: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('/data/Yeonju/commentData.json', {
+      method: 'GET',
+    })
```

Comment:
- fetch 함수의 기본 메서드는 get 이기 때문에, get 메서드의 경우 option 부분을 생략해줄 수 있습니다.
```suggestion
    fetch('/data/Yeonju/commentData.json')
```

Reply:
넵!